src/components/ErrorBanner.jsx

```diff
@@ -1,9 +1,16 @@
-import React from 'react'
+import React from "react";
 
-const ErrorBanner = () => {
+function ErrorBanner() {
   return (
-    <div>ErrorBanner</div>
-  )
+    <div
+      className=" flex justify-center items-center mb-3 rounded-md w-auto
+     h-12 bg-[#FFB1B1] shrink-0 "
+    >
+      <p className="text-[#991B1B] text-[15px] text-center font-Rubik font-normal not-italic leading-4  ">
```

Comment:
you don't have to set the font, that is already done globally on the app. 

If anything is "normal" you don't have to set that, for example we have the font-normal not-italic classes here, but they are not changing anything.

Reply:
most of the things really didn't change the design. I was just following the figma code :)